### PR TITLE
SOA RDM Spell Corrections

### PR DESCRIPTION
**Pull request by neuromancerxi**
Corrected Animations and Groups for Distract/II,Flurry/II,Frazzle/II.
Added Distract II as usable item.
Created item script for Scroll of Distract II.
Changed Relevant stats from MND to INT for Distract/II, and Frazzle/II.

Tested against Stable Branch, Client Version 30171205_0.

Please see cb7753a for Video Reference URLs for Animations from Retail.

### CODE REVIEW

File: scripts/globals/spells/distract.lua [hunk 1]
```diff
@@ -13,7 +13,7 @@ end;
 function onSpellCast(caster,target,spell)
 
     -- Pull base stats.
-    local dMND = (caster:getStat(MOD_MND) - target:getStat(MOD_MND));
+    local dINT = (caster:getStat(MOD_INT) - target:getStat(MOD_INT));
```
Comment (TeoTwawki) [inline scripts/globals/spells/distract.lua]: Sorry but despite being black magic its MND based

https://www.bg-wiki.com/bg/Distract
https://www.bg-wiki.com/bg/Distract_III

```
This spell reduces the target's evasion based on dMND and Enfeebling Skill with this formula:

    Potency = [Floor(6/21)(Enfeebling Skill - 190)] + Floor(dMND/5); where dMND/5 caps at 0 and 10.
        Lowest value at ±0 dMND compared to enemy: 0
        Highest value at +50 dMND compared to enemy: 10
            When Enfeebling Magic skill is 610, players will reach the highest value of -120 evasion. The value from MND above will be added to this.
                Caps at -130 evasion.
```

File: scripts/globals/spells/distract_ii.lua [hunk 2]
```diff
@@ -11,12 +11,12 @@ function onMagicCastingCheck(caster,target,spell)
 end;
 
 function onSpellCast(caster,target,spell)
-    local dMND = (caster:getStat(MOD_MND) - target:getStat(MOD_MND));
-    local power = utils.clamp(40+math.floor(dMND/5), 40, 50);
+    local dINT = (caster:getStat(MOD_INT) - target:getStat(MOD_INT));
```
Comment (TeoTwawki) [inline scripts/globals/spells/distract_ii.lua]: see comment on 1st tier of same spell line

File: scripts/globals/spells/frazzle_ii.lua [hunk 3]
```diff
@@ -11,12 +11,12 @@ function onMagicCastingCheck(caster,target,spell)
 end;
 
 function onSpellCast(caster,target,spell)
-    local dMND = (caster:getStat(MOD_MND) - target:getStat(MOD_MND));
-    local power = utils.clamp(40+math.floor(dMND/5), 40, 50);
+    local dINT = (caster:getStat(MOD_INT) - target:getStat(MOD_INT));
+    local power = utils.clamp(40+math.floor(dINT/5), 40, 50);
```
Comment (TeoTwawki) [inline scripts/globals/spells/frazzle_ii.lua]: https://www.bg-wiki.com/bg/Frazzle_II
```
Grants magic evasion down based on the casters Enfeebling Magic skill and the differential in caster MND to target MND.

    Lowest value at ±0 dMND compared to enemy: 0
    Highest value at +50 dMND compared to enemy: 10
    When enfeebling magic skill is 350, players will reach the highest value of -40 magic evasion. The value from MND above will be added to this.
        Caps at -50 magic evasion.
```

**Comment by neuromancerxi** [comment]
Still noobing around with PR's, hope I'm doing this right. Please see 3da1de6 for reversion to MND instead of INT.

**Comment by TeoTwawki** [comment]
:+1:

**Pull request merged**